src/main/java/it/gov/pagopa/payhub/activities/service/classifications/trclassifiers/TransferClassifier.java

```diff
@@ -42,17 +42,27 @@ default Long getAmountCents(Transfer transferDTO) {
 	 * @param paymentsReportingDTO the payment reporting data.
 	 * @return the amount in cents, or {@code null} if the amount is not available.
 	 */
-	default Long getAmountCents(PaymentsReporting paymentsReportingDTO) {
+	default Long getAmountPaidCentsFromPaymentsReporting(PaymentsReporting paymentsReportingDTO) {
```

Comment:
Why not the following?
```suggestion
	default Long getTransferAmountCents(PaymentsReporting paymentsReportingDTO) {
```